lib/PayPal/Converter/FormatConverter.php

```diff
@@ -1,62 +0,0 @@
-<?php
```

Comment:
This class should probably be kept in case someone was using it (e.g. no major revision changes). Maybe mark it deprecated?